### ISSUE TITLE
DAOS: higher scale test with verbs times out with connection timeout errors

### ISSUE THREAD

**Issue by frostedcmos** [ofiwg/libfabric#7124]
Setup: 16 servers, 64 clients, 32 process per node
OFI tested: e7b4dccbc9d194965b60707d3689c73ab94eb2ae
Provider: verbs;ofi_rxm

When ran at this scale on TACC, some clients end up timing out with following libfabric error: "fi_eq_readerr: err: Connection timed out" and ior tests hang.

Similar problems were identified with a different scale test of 1 server/engine, 50 clients, 32ppn

**Comment by shefty** [comment]
Do we have the logs available for when this occurs?

**Comment by shefty** [comment]
This issue was addressed by #7202.  Connection requests that timeout are now retried.  The timeouts themselves are coming from SA path record queries, which is common at this scale.

Testing has scaled successfully once to 45 nodes, 32 ppn.  The results are unreliable.  At 48 nodes, 16 ppn, we were able to run multiple times.  Above that, we hit DAOS abort errors of some sort that we've been unable to identify any cause for -- not even sure it's in libfabric.

Closing as original issue is resolved.  Is DAOS known to scale above 48 nodes, 16 ppn?
